### PR TITLE
Feature/init kangal jmeter

### PR DESCRIPTION
**Pull request by s-radyuk**
This PR adds Kangal-JMeter content:
- Docker images for building JMeter master and worker 
- Launcher.sh script running JMeter process in JMeter master and saving report to AWS
- Additional files required for building images: log4j, MySQL connector, security keys
- README.md with images description

### CODE REVIEW

**Comment by fftorres** [comment]
+1

File: docker/jmeter-master/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+ENV SSL_DISABLED false
+ENV WORKER_SVC_NAME jmeter-worker
+ENV TESTS_DIR /tests
+ENV SLEEP 5
+ENV USE_WORKERS false
+
+ENV AWS_DEFAULT_REGION ""
+ENV AWS_ENDPOINT_URL ""
+ENV AWS_BUCKET_NAME ""
```
Comment (aleksandrzhiliaev) [inline docker/jmeter-master/Dockerfile:13]: Please leave some comments why do we need this envs

Comment (s-radyuk) [reply docker/jmeter-master/Dockerfile:13]: Added info to the README

**Comment by s4nji** [comment]
+1

**Comment by RTodorov** [comment]
+1

**Pull request merged**